README.md

```diff
@@ -1,2 +1,3 @@
 # second_repository
 My second repository made for homework
+Open visual studio and press F5
```

Comment:
You have very little information in the README.md file, add a description of your task, and better explain how to run your program